### PR TITLE
updated file structure

### PR DESCRIPTION
**Pull request by archit7448**
Capture (2)

capture 2

### What to Check 

- [X] Data Context

- [x]  Categories js

- [x] card CSS

### What Not To Check 

- [ ]  CSS Part reviewed before

### CODE REVIEW

File: src/pages/Homepage/Homepage.css [hunk 1]
```diff
@@ -0,0 +1,40 @@
+@import url("https://fonts.googleapis.com/css?family=Special+Elite");
+blockquote {
+  font-family: "Special Elite", cursive;
+  font-size: 3rem;
+}
+.homepage-main {
+  background: #fffdf5;
```
Comment (zoef-yt) [inline src/pages/Homepage/Homepage.css:7]: use colour var if you have created it.

**Review by zoef-yt — APPROVED** [review]
The code is good. Nice use of useReduce. Code is good to merge.

File: src/context/Data.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import axios from "axios";
```
Comment (sudipkundu999) [inline src/context/Data.js:1]: Appreciation: The logic is very well written. Good use of all concepts.

**Review by sudipkundu999 — APPROVED** [review]
Good UI and easy to understand code.

**Pull request merged**